src/main/java/at/hannibal2/skyhanni/config/features/crimsonisle/CrimsonIsleConfig.kt

```diff
@@ -44,6 +44,15 @@ class CrimsonIsleConfig {
     @FeatureToggle
     var pabloHelper: Boolean = false
 
+    @Expose
+    @ConfigOption(
+        name = "Sirih NPC Helper",
+        desc = "Show a clickable message that grabs sulphur from your sacks."
+    )
+    @ConfigEditorBoolean
+    @FeatureToggle
+    var sirihHelper: Boolean = false
```

Comment:
```suggestion
    var sirihHelper: Boolean = true
```

its very unintrusive, can absolutely be default enabled

Reply:
pablo helper wasnt enabled by default so i figured this shouldnt as well

Reply:
Feel free to enable Pablo helper per default in your pr